ovs/extensions/services/service.py

```diff
@@ -15,14 +15,20 @@
 """
 Service Factory module
 """
+import time
 
-from subprocess import check_output
+from subprocess import check_output, CalledProcessError
 from ovs.extensions.services.upstart import Upstart
 from ovs.extensions.services.systemd import Systemd
 from ovs.log.logHandler import LogHandler
 
 logger = LogHandler.get('extensions', name='servicemanager')
-
+try:
+    from ovs.extensions.services.fleetctl import FleetCtl
```

Comment:
This try except is only interesting for systemd ==> move it in the else stack of systemd, no need to try to import on ubuntu